framework/src/main/java/org/checkerframework/framework/type/visitor/AnnotatedTypeCombiner.java

```diff
@@ -39,7 +39,7 @@ public static void combine(
      *
      * @param hierarchy the hierarchy used to the compute the GLB
```

Comment:
Mentions of GLB and "greatest lower bound" in the documentation need to be generalized, just as the code has been generalized.